packages/venia-concept/src/CustomButton/CustomButton.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { oneOf, shape, string, bool } from 'prop-types';
+
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+import defaultClasses from '@magento/venia-ui/lib/components/Button/button.css';
+
+import useThemedCss from '@magento/venia-concept/src/ThemedCss/useThemedCss';
+
+const getRootClassName = (priority, negative) =>
+    `root_${priority}Priority${negative ? 'Negative' : ''}`;
+
+const CustomButton = props => {
+    const {
+        children,
+        classes: propClasses,
+        priority,
+        type,
+        negative,
+        disabled,
+        className,
+        ...restProps
+    } = props;
+
+    // One of two changes to "Button". Wrap "defaultClasses" in `useThemedCss`.
+    const classes = mergeClasses(
+        useThemedCss('Button', defaultClasses),
+        propClasses
+    );
+    const rootClassName = classes[getRootClassName(priority, negative)];
+
+    return (
+        <button
+            // The only other custom thing here...But there might be a better
+            // way to ensure that incoming classes are used.
+            className={`${className} ${rootClassName}`}
```

Comment:
I think it would be preferable to just modify the `Button` component itself with this and the changes from line 25. No copy of `Button` would be necessary, just need to import the hook and pass it the component name + default classes.